packages/lodestar/test/spec/allForks/forkChoice.ts

```diff
@@ -208,7 +215,11 @@ function runStateTranstion(
     justifiedBalances = getEffectiveBalances(justifiedState);
   }
   try {
-    forkchoice.onBlock(signedBlock.message, postState, {justifiedBalances});
+    forkchoice.onBlock(signedBlock.message, postState, {
+      justifiedBalances,
+      justifiedActiveValidators,
+      justifiedTotalActiveBalanceByIncrement,
+    });
```

Comment:
I wonder how does the spec test specify `blockReceptionTime`?

Reply:
there is a `step` for update time here: https://github.com/ChainSafe/lodestar/pull/3540/files#diff-49a2a7fb7007eb44eff9a66634a30ff5475e4529132d81b6960675a86b98e1b3R57-R58